pages/blog/migrating-from-rest.mdx

```diff
@@ -0,0 +1,133 @@
+      }).then(res => res.json()),
+    updateUser:  (root, args) => fetch('https://myrest.com/user' + args.id, {
+       method: 'POST',
+       body: JSON.stringify(args.input)
+    }).then(res => res.json()),
+    deleteUser:  (root, args) => fetch('https://myrest.com/user' + args.id, {
+       method: 'DELETE'
+    }).then(res => res.json()),
+};
+```
+
+This example assumes that you have `/user/:id` endpoint that gets a `User` entity with `HTTP GET`, deletes user with `HTTP DELETE` and updates a `User` that has the given id with the given input. Also `/user` endpoint creates a new `User` with the given input.
+
+But this implementation will be hard to maintain when the REST API is updated and become bigger.
+
+### Using GraphQL Mesh instead without any code
+
+GraphQL Mesh is a tool that handles multiple non-GraphQL data sources and generates an executable GraphQL schema on top of them with a simple configuration file.
+
+It has JSON Schema handler that generates a GraphQL Schema based on the given JSON schema files. This handler can also generate JSON Schema on runtime based on the given sample request and response data.
+
+First you need to create a project using `yarn` on an empty directory:
+
+```bash
+yarn init
+```
+
+After that we need to install some dependencies of Mesh:
+
+```bash
+ yarn add @graphql-mesh/cli @graphql-mesh/json-schema graphql
+```
+
+Create a `.meshrc.yml` which is a configuration file for GraphQL Mesh on our new project:
+
+```yml
+sources:
+  - name: MyRest
+    handler:
+      jsonSchema:
+        baseUrl: https://myrest.com/
+        operations:
+          - type: Query
+            field: user
+            path: /user/{args.id}
+            method: GET
+            responseSample: ./getUserResponse.json
+          - type: Mutation
+            field: createUser
+            path: /user
+            method: PUT
+            requestSample: ./createUserRequest.json
+            responseSample: ./createUserResponse.json
+          - type: Mutation
+            field: updateUser
+            path: /user/{args.id}
+            method: POST
+            requestSample: ./updateUserRequest.json
+            responseSample: ./updateUserResponse.json
+          - type: Mutation
+            field: deleteUser
+            path: /user/{args.id}
+            method: DELETE
+            responseSample: ./deleteUserResponse.json
+```
+
+As you can see in the configuration, we defined our endpoints without a single line of code. After creating this configuration file. We need to get sample request and response files by calling those endpoints on our local.
+
+With a single command, our new GraphQL endpoint is ready to serve;
+
+```bash
+$ yarn mesh serve
+```
+
+### Not only as a Gateway but also a completely type-safe SDK
+
+Mesh is able to generate a type-safe SDK from generated GraphQL API because the generated GraphQL schema is a local `GraphQLSchema` that can be executed without binding an HTTP server.
```

Comment:
Add:

That means you can use GraphQL Mesh inside your existing services or clients, [as an SDK](link to this part of the docs), just as a simple dependency, without adding another box in your architecture.